free-2.md

```diff
@@ -5,51 +5,93 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL for keeping track of appointments across many different devices easily.
```

Comment:
At first glance, this seems like it is already a bit on the small side. There are a lot of applications and projects that already meet similar goals. A good version might be a way to add appointments to a calendar in a single way that supports many backends, i.e. google calendar, outlook, ical, etc.